src/fprime_gds/common/loaders/xml_loader.py

```diff
@@ -218,14 +219,20 @@ def get_enum_type(self, enum_name, xml_obj):
         for enum in enum_section:
             # Check enum name
             if enum.get(self.ENUM_TYPE_TAG) == enum_name:
+                # Get serialize/representation type, if present
+                serialize_type = enum.get(self.ENUM_SERIALIZE_TYPE_TAG)
```

Comment:
Can you add a default in this call? e.g.:

```
serialize_type = enum.get(self.ENUM_SERIALIZE_TYPE_TAG, "I32")
```

I don't remember if this will work here or not, but most python get calls support this.

Reply:
This seems to work OK.